### PR TITLE
refactor: remove FULL-SYNC-CUT cmd #2687

### PR DESCRIPTION
**Pull request by BorysTheDev**

### CODE REVIEW

File: src/server/journal/streamer.cc [hunk 1]
```diff
@@ -65,39 +65,38 @@ void RestoreStreamer::Start(io::Sink* dest) {
...
-    snapshot_finished_ = true;
-  });
+    if (last_yield >= 100) {
+      ThisFiber::Yield();
+      last_yield = 0;
+    }
+  } while (cursor);
+}
+
+void RestoreStreamer::SendFullSyncCut() {
```
Comment (adiholden) [inline src/server/journal/streamer.cc:94]: I am confused.. are you still using this?

Comment (BorysTheDev) [reply src/server/journal/streamer.cc:94]: I forget to remove it. It was intermediate step during refactoring. thx

File: src/server/cluster/outgoing_slot_migration.cc [hunk 2]
```diff
@@ -17,46 +18,58 @@ class OutgoingMigration::SliceSlotMigration {
...
+  MigrationState state = MigrationState::C_NO_STATE;
+  {
+    std::lock_guard lck(flows_mu_);
+    slot_migrations_[shard_id] =
+        std::make_unique<SliceSlotMigration>(slice, slots_, sync_id, journal, &cntx_, dest);
+    state = GetStateImpl();
+  }
+
+  if (state == MigrationState::C_FULL_SYNC) {
+    main_sync_fb_ = Fiber("outgoing_migration", &OutgoingMigration::SyncFb, this);
```
Comment (adiholden) [inline src/server/cluster/outgoing_slot_migration.cc:71]: why do you need the main sync fb?

File: tests/dragonfly/cluster_test.py [hunk 3]
```diff
@@ -1090,28 +1063,10 @@ async def generate_config():
...
-
-        await asyncio.sleep(0.1)
-
-    # Give seeder one more second
-    await asyncio.sleep(1.0)
-
     # Stop seeder
     seeder.stop()
     await fill_task
+    await asyncio.sleep(1.0)
```
[the remark this answers is not in the document]
Comment (BorysTheDev) [reply tests/dragonfly/cluster_test.py:1069]: I can add one more, I need this to be sure that migration is finished

File: src/server/cluster/outgoing_slot_migration.cc (hunk 2 above)
Comment (BorysTheDev) [reply src/server/cluster/outgoing_slot_migration.cc:71]: To know when snapshot sending is finished. After that I can initiate Finalize cmd automatically

**Review by adiholden — APPROVED** [review]

**Pull request merged**